docs/excel/custom-functions-overview.md

```diff
@@ -175,19 +175,19 @@ function incrementValue(increment, setResult){
 }
 ```
 
-For streamed functions, the final parameter, setResult, is never specified in your registration code, and it does not display in the autocomplete menu to Excel users when they enter the function. It’s a callback function that’s used to pass data from the function to Excel to update the value of a cell. In order for Excel to pass the setResult function, you must declare support for streaming during your function registration by setting the parameter stream to true.
+For streamed functions, the final parameter, `setResult`, is never specified in your registration code, and it does not display in the autocomplete menu to Excel users when they enter the function. It’s a callback function that’s used to pass data from the function to Excel to update the value of a cell. In order for Excel to pass the `setResult` function, you must declare support for streaming during your function registration by setting the parameter `stream` to `true`.
 
 ## Saving state
 
-Custom functions can save data in global JavaScript variables. In subsequent calls, your custom function may use the values saved in the global variables. Saved state is useful when users enter multiple instances of the same custom function, and each instance needs to share the same data. For example, you may save the data returned from a call to a web resource to avoid making additional calls to the same web resource.
+Custom functions can save data in global JavaScript variables. In subsequent calls, your custom function may use the values saved there. Saved state is useful when users enter multiple instances of the same custom function, and they need to share data with each other. For example, you may save the data returned from a call to a web resource to avoid making additional calls to the same web resource.
```

Comment:
@saunders77 can you change "values saved there" to "values saved in these variables." Thanks!